Cython/Compiler/PyrexTypes.py

```diff
@@ -4368,6 +4374,72 @@ def same_as_resolved_type(self, other_type):
     def error_condition(self, result_code):
         return "dummy"
 
+class InitOrClassVar:
+    """Used to help Cython interpret dataclass InitVar or ClassVar.
+
+    Although not really a CppClassType, it uses a template-like syntax
+    so claims to be to allow exisiting code to use it with minimal fuss
```

Comment:
I had to read this sentence three times in order to make it pass my parser.
```suggestion
    Although not really a CppClassType, it uses a template-like syntax.
    So it mimics the interface to let existing code use it with minimal fuss.
```